terraform/modules/shared-infrastructure/user_data.sh

```
@@ -0,0 +1,65 @@
+#!/bin/bash
+yum update -y
+yum install -y amazon-efs-utils
+
+# Configure ECS agent
+echo "ECS_CLUSTER=${cluster_name}" >> /etc/ecs/ecs.config
```

Shellcheck (0.10.0)

[warning] cluster_name is referenced but not assigned.

(SC2154)